admin/partials/perfecty-push-admin-post-metabox.php

```diff
@@ -1,6 +1,20 @@
-<input type="checkbox"
+<p><input type="checkbox"
 	   id="perfecty_push_send_on_publish"
 	   name="perfecty_push_send_on_publish"
 	   <?php if ( $send_notification === true ) {
 		   echo 'checked="checked"';} ?>"/>
-<label for="perfecty_push_send_on_publish"><?php printf( esc_html__( 'Send notification on publish', 'perfecty-push-notifications' ) ); ?></label>
+<label for="perfecty_push_send_on_publish"><?php printf( esc_html__( 'Send notification on publish', 'perfecty-push-notifications' ) ); ?></label></p>
+<p><input type="checkbox"
+	   id="perfecty_push_send_featured_img"
+	   name="perfecty_push_send_featured_img"
+	   <?php if ( $send_featured_img === true ) {
+		   echo 'checked="checked"';} ?>"/>
+<label for="perfecty_push_send_featured_img"><?php printf( esc_html__( 'Send featured img in payload', 'perfecty-push-notifications' ) ); ?></label></p>
+<p>&nbsp;</p>
+<p><label for="perfecty_push_notification_custom_title"><?php printf( esc_html__( 'Custom notification title (leave blank for blogname)', 'perfecty-push-notifications' ) ); ?></label>
```

Comment:
Cosmetic: "Custom notification title (leave blank for blogname)" -> Could we use "Custom title (default: website name)"?